pinot-controller/src/main/java/org/apache/pinot/controller/recommender/rules/io/configs/IndexConfig.java

```diff
@@ -80,6 +80,10 @@ public boolean isSortedColumnOverwritten() {
         return _isSortedColumnOverwritten;
     }
 
+    public void setSortedColumnOverwritten(boolean sortedColumnOverwritten) {
```

Comment:
This should not be needed. Already covered in 

```
@JsonSetter(nulls = Nulls.SKIP)
    public void setSortedColumn(String sortedColumn) {
        this._sortedColumn = sortedColumn;
        this._isSortedColumnOverwritten = true;
    }
```

Reply:
Usually deserializers require standard setters to be able to deserialize to objects. We didn't need this before, because `RecommenderDriver.run(input)` returns a string and that's only used in the endpoint. In tests, the output of each rule is begin tested. Now for the RT.Prov.Rule, the new test verify against the output of the run function - which is more comprehensive - and that requires deserialization of the output string.